src/main/java/pl/cleankod/exchange/core/domain/Account.java

```diff
@@ -12,10 +14,17 @@ public static record Id(UUID value) {
             Preconditions.requireNonNull(value);
         }
 
+        @Override
+        @JsonValue
```

Comment:
We polluted our core domain with framework-specific stuff. This also begs the question about the core domain being exposed via our external API. I smell lots of trade-offs - can you spot them? Any ideas on how to approach that?

Reply:
Jackson specifically have MixIns concept, which is often used on library code serialization that we don't have access to.

Reply:
On exposing domain classes to API, not a good idea at all, often domain classes have different structure than REST API model. I would opt to create specific controller domain pojos and introduce Mapstruct into the project.